app/src/main/java/org/mozilla/fenix/wallpapers/WallpaperManager.kt

```diff
@@ -134,6 +142,28 @@ class WallpaperManager(
         }
     }
 
+    fun recordDiscoveredMetric() {
+        val hasSentMetric = settings.wallpapersDiscovered
+        if (!hasSentMetric) {
```

Comment:
Run-by: Perhaps drop `hasSentMetric` and inline `settings.wallpapersDiscovered`. I would normally consider using a variable if the conditional was much more complex, but this is a fairly simple conditional.

Reply:
Removed as part of data review